packages/schemix/src/schemix/sqlite/columns.py

```
@@ -0,0 +1,132 @@
+
+
+class Numeric(
+    PassthroughSerializationMixin,
+    NumericOperatorMixin,
+    OrderedOperatorMixin,
+    CountOperatorMixin,
+    MinMaxOperatorMixin,
+    NumericAggregateOperatorMixin,
+    ColumnType[float],
+):
+    def __init__(self, col_name: str, *, precision: int, scale: int = 0) -> None:
```

Pylint (3.3.7)

[warning] Useless parent or super() delegation in method '__init__'

(W0246)